src/labkey/query/Rows.ts

```diff
@@ -13,51 +13,75 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { request, RequestOptions } from '../Ajax'
-import { IFilter } from '../filter/Filter'
+import { request } from '../Ajax'
 import { buildURL } from '../ActionURL'
-import { ExtendedXMLHttpRequest, getCallbackWrapper, getOnFailure, getOnSuccess, isArray } from '../Utils';
-import { buildQueryParams, ContainerFilter, getMethod, getSuccessCallbackWrapper } from './Utils'
+import { getCallbackWrapper, getOnFailure, getOnSuccess, RequestCallbackOptions } from '../Utils';
+
+export interface QueryRequestOptions extends RequestCallbackOptions {
+    apiVersion?: number | string
+    /**
+     * The container path in which the schema and query name are defined.
+     * If not supplied, the current container path will be used.
+     */
+    containerPath?: string
+    /** **Experimental:** Optional extra context object passed into the transformation/validation script environment. */
+    extraContext?: any
+    /**
+     * Name of a query table associated with the chosen schema.
+     * See also: [How To Find schemaName, queryName & viewName](https://www.labkey.org/Documentation/wiki-page.view?name=findNames).
+     */
+    queryName: string
+    rowDataArray?: any[]
+    /**
+     * Array of record objects in which each object has a property for each field.
+     * The row data array needs to include only the primary key column value, not all columns.
+     */
+    rows?: any
```

Comment:
type as `any[]`?

Reply:
Switch back to `any[]`. Made appropriate update in `@labkey/components` PR.